ddtrace/contrib/django/conf.py

```diff
@@ -34,6 +34,7 @@
     'DEFAULT_DATABASE_PREFIX': '',
     'DEFAULT_SERVICE': 'django',
     'ENABLED': True,
+    'DISTRIBUTED_TRACING': True,
```

Comment:
I think it should be disabled by default? otherwise you're always listening to tracing headers even if it's a public facing service, so people can inject fake `trace_id` during each request. In other services we're keeping distributed tracing off by default.